ruby/ql/test/library-tests/dataflow/summaries/summaries.rb

```diff
@@ -82,4 +85,22 @@ def userDefinedFunction(x, y)
 a.withoutElementOne()
 sink(a[0])
 sink(a[1])
-sink(a[2]) # $ hasValueFlow=elem2
\ No newline at end of file
+sink(a[2]) # $ hasValueFlow=elem2
+
+x = Foo.new
+y = []
+z = []
+x.flowToAnyArg(tainted, y, key: z)
```

Comment:
Could you add similar calls to `sink()` before the call to `flowToAnyArg`?

Reply:
Done :+1: